doc/update/README.md

```diff
@@ -13,10 +13,12 @@ configured the packages to run this command automatically after the new package
 is installed. If you are installing GitLab 7.9 or earlier, please see the
 [procedure below](#updating-from-gitlab-66-and-higher-to-the-latest-version).
 
-First, download the latest [CE](https://packages.gitlab.com/gitlab/gitlab-ce) or
+If you installed using the package server all you need to do is run `sudo apt-get update && sudo apt-get install gitlab-ce` (for Debian/Ubuntu) or `sudo yum install gitlab-ce` (for CentOS/Enterprise Linux).
```

Comment:
Oh yeah, I hadn't payed attention what the site said. Better be consistent, don't change anything for the time being. Thanks